frontend/lib/laletterbuilder/homepage.tsx

```diff
@@ -115,12 +115,13 @@ export const LaLetterBuilderHomepage: React.FC<{}> = () => (
     </section>
     <section className="jf-laletterbuilder-landing-section-primary">
       <div className="hero-body">
-        <div className="container jf-has-text-centered-tablet">
+        <div className="jf-laletterbuilder-issues-list container jf-has-text-centered-tablet">
```

Comment:
if we're going to use this class for FAQs and issues (which I think we should, that's great) maybe change the name of the class to `jf-accordion-list` or something?

Also in case you were wondering, the `jf` convention is intended to distinguish between overrides of Bulma classes (the CSS framework we use) and classes we wrote ourselves. Not sure if it's 100% applied across the codebase, but that's the idea.

Reply:
ooh good to know (& will update the name)! not related to this pr but what's the convention for starting files with an underscore?